Core/include/Acts/Seeding/SeedFinderFTFConfig.hpp

```diff
@@ -57,12 +57,24 @@ struct SeedFinderFTFConfig {
   bool m_LRTmode = true;  // eventually want to set from full chain
   bool m_useEtaBinning = true;
   bool m_doubletFilterRZ = true;
-  float m_minDeltaRadius = 5.0;  // eventually set in config or to equivalent
-                                 // acts 2.0 but increasing to test loops
-  // float m_maxDeltaRadius = 270.0 ;
-  float m_tripletD0Max = 4.0;  // m_settings
+  float m_minDeltaRadius = 2.0;  // athena itk 2.0
+  float m_tripletD0Max = 4.0;    // m_settings
   unsigned int m_maxTripletBufferLength = 3;
 
+  // from Run function //used to be const but can't as FTF config needs to be
+  // changeable
+  int MaxEdges = 2000000;
+  float cut_dphi_max = 0.012;
+  float cut_dcurv_max = 0.001;
+  float cut_tau_ratio_max = 0.007;
+  float maxOuterRadius = 550.0;
+  float m_PtMin = 1000.0;
+  float m_tripletPtMinFrac = 0.3;
+  float m_tripletPtMin =
+      m_PtMin *
+      m_tripletPtMinFrac;  // used to be 2500, changing this helps low pt a lot
+  double ptCoeff = 0.29997 * 1.9972 / 2.0;  // from athena
+
```

Comment:
Thank you for this PR! could you add some comments here in the config file explaining what are all of these variables?

Reply:
Thank you for reviewing! I have added in descriptions